support-frontend/assets/components/orderSummary/contributionsOrderSummaryContainer.tsx

```diff
@@ -45,10 +59,38 @@ export function ContributionsOrderSummaryContainer({
 }: ContributionsOrderSummaryContainerProps): JSX.Element {
 	const contributionType = useContributionsSelector(getContributionType);
 
-	const { currencyId } = useContributionsSelector(
+	const { countryId, currencyId } = useContributionsSelector(
 		(state) => state.common.internationalisation,
 	);
-	const selectedAmount = useContributionsSelector(getUserSelectedAmount);
+	const { productType } = useContributionsSelector(
+		(state) => state.page.checkoutForm.product,
+	);
+	const billingPeriod = (productType[0] +
+		productType.slice(1).toLowerCase()) as BillingPeriod;
+	const currentPrice = useContributionsSelector(getUserSelectedAmount);
+	const originalPrice = useContributionsSelector((state) =>
+		getOriginalPrice(
+			state.page.checkoutForm.product.productPrices,
+			countryId,
+			billingPeriod,
+		),
+	);
+
+	function getOriginalPrice(
+		productPrices: ProductPrices,
+		country: IsoCountry,
+		billingPeriod: BillingPeriod,
+		fulfilmentOption: FulfilmentOptions = NoFulfilmentOptions,
+		productOption: ProductOptions = NoProductOptions,
+	): number | undefined {
+		const countryGroup = getCountryGroup(country);
+		const originalPrice =
+			productPrices[countryGroup.name]?.[fulfilmentOption]?.[productOption]?.[
+				billingPeriod
+			]?.[countryGroup.currency]?.price ?? 0;
+		return originalPrice > currentPrice ? originalPrice : undefined;
+	}
```

Comment:
I'd really like a test here of sorts to describe this (not sure we have tests for this type of stuff). But if I am reading this right...

1. We have a `currentPrice` - which will be the price with a promotion applied.
2. This might be different to the price in the catalogue, which is stored in `productPrices`
3. We only do this on S+ as contributions technical price is `0`
4. We send over `originalPrice` only if it's different and use that optionality to render differently

On point 4 - I wonder if we always send current + original price and let the rendering component calculate what to do with that?

Reply:
Hmm, I thought that but thought it looked more messy. The `originalPrice` (to be later `productPrice`) is only returned if its available AND is greater than the `currentPrice` (to be later`productPriceWithPromo`) otherwise its `undefined`. erhaps we can cleanup further if needed after this PR.